src/python/pants/option/config_test.py

```diff
@@ -43,6 +43,7 @@ class ConfigFile:
 
         [b]
         preempt = true
+        embedded = "This is a path: %(embed)s"
```

Comment:
Added this just to be really sure things still work with multiple cascading interpolations inside and outside the DEFAULT section.